### ISSUE TITLE
[feature request] user list bar

### ISSUE THREAD

**Issue by LordMZTE** [osa1/tiny#256]
I'd love to have an option to show a list of users on the right side of the TUI similar to the user list many IRC clients show.

Such as this user list in HexChat (outlined in red)
![image](https://user-images.githubusercontent.com/28735087/93516144-e76ee000-f929-11ea-88a4-bfc4b513080b.png)

Maybe this could be toggled similar to `/statusline` using a command (like `/userlist` for example)

**Comment by osa1** [comment]
This is definitely planned, we discussed this many times on the IRC channel, but it's a bit too much work for me to implement currently, so don't expect it in the next few weeks.

**Comment by osa1** [comment]
@trevarj implemented a list widget in https://github.com/osa1/tiny/discussions/284 which could be used to implement user list too.

As far as I remember, the problems were:

- Side bar makes copying text difficult. Multi-line text cannot be copied without closing the list first.
- We would need to find some key bindings (maybe modal) to scroll in the list, select an entry etc.

**Comment by trevarj** [comment]
Yes, just to update, I really feel like having "dummy" tabs can solve this issue nicely. For example, `/names` can forward its output to a new tab and then you can scroll through it. Maybe we can even have it be an active tab, which will update and notify on change (nick joining)

This would keep a clean/minimal layout.

Anyway, just brainstorming. I can be probably be convinced to be in favor of the list widget again...
